theme.json

```diff
@@ -509,7 +545,13 @@
 			}
 		},
 		"spacing": {
-			"blockGap": "var:custom|spacing|small"
+			"blockGap": "var:custom|spacing|small",
```

Comment:
Should we use this notation, or the `--wp--preset--spacing..` nomination etc (for consistency)?

Also, should we deprecate custom|spacing|small (maybe add a CSS fallback?) and use the spacing numeral values from spacingSizes instead?

Reply:
Yes, exactly. 
I am thinking of using `--wp--preset--spacing...` and adding CSS fallback for --wp--custom--spacing--small with --wp--preset--spacing--30(both have same size). 

The idea is to keep theme.json up to date.  Add CSS fallback to deprecate elements.